apis/sgv2-docsapi/src/main/java/io/stargate/sgv2/docsapi/api/v2/namespaces/collections/documents/model/dto/BuiltInFunctionDto.java

```diff
@@ -27,12 +27,15 @@
 /** DTO for a namespace. */
 @JsonInclude(JsonInclude.Include.NON_NULL)
 public record BuiltInFunctionDto(
-    @Schema(description = "The operation to execute", pattern = "\\$pop|\\$push", example = "$push")
+    @Schema(
+            description = "The operation to execute",
+            pattern = "\\$pop|\\$push|\\$set",
+            example = "$push")
         @NotNull(message = "`operation` is required for function execution")
         @NotBlank(message = "`operation` is required for function execution")
         @Pattern(
-            regexp = "\\$pop|\\$push",
-            message = "available built-in functions are $pop and $push")
+            regexp = "\\$pop|\\$push|\\$set",
```

Comment:
we should add a body example for the new function in the api documentation, same way we have for the pop and push